### PR TITLE
Adds a new ruin: The bombed airbase

### PR DESCRIPTION
**Pull request by Erikafox**
## About The Pull Request
Adds a new ruin, the bombed airbase. An overgrown leftover from the ICW, this base was struck from orbit and quickly abandoned by its survivors. Time has flown and the base is now overgrown by nature, the wounds of orbital bombardment and excessive pouring of concrete slowly healing.

Loot list:
10 plasma cans, 2 oxygen cans
2 pilot suits 
3 atmos firesuits
3 backpack firefighter tanks
1 APLU 3 Firefighter
2 mech extinguishers
2 hydraulic clamps
32 pml rockets (good luck using these)
Kitchen stuff
2 Stechkins
4 Stechkin Magazines
Combat Uniforms and Combat Boots
ACLF uniforms
About half of a telecomms set up
1 binoculars
Wideband intercomm
4x plasma engines and heaters (intact)
3x RTGs
Some donk pockets
Cans of beans
Metal and Glass (about 40 count)

This is a fairly hefty list, and this ruin is not without its hazards. Those who approach too close to where the base was hit will find themselves becoming irradiated, and diving into the densely done part of the base will bring the spiders who have moved into the base into a fight. It's best to be careful while approaching this place.

- [ ] I affirm that I have tested all of my proposed changes and that any issues found during tested have been addressed.

## Why It's Good For The Game

It's another big ruin with some interesting variety.
## Changelog

:cl:
add: The Bombed Airbase ruin, filled with scrap, spiders, and starfighters.
/:cl:

### CODE REVIEW

**Comment by Erikafox** [comment]
I couldn't get the ruin to spawn. The process started and I walked off for a good half an hour before coming back, and it was still trying to spawn. So I may need some help with that.
edit: In what is sure to become a reoccurring theme. My stuff was just broken and it all works fine now.

File: code/game/area/areas/ruins/jungle.dm [hunk 1]
```diff
@@ -96,3 +96,19 @@
 	name = "cargo dump"
 	icon_state = "dk_yellow"
 
+// Bombed Syndicate Starport
```
Comment (Latentish) [inline code/game/area/areas/ruins/jungle.dm:108]: Please limit this down to 2 or less unique areas.

**Comment by Latentish** [comment]
While I still need you to do the biz with cutting your areas down, I'd like to tm this if the checks pass.

**Comment by Latentish** [comment]
I adore this ruin. Finally going in and testing it, it's awesome. That being said there's a few issues.
- For the challenge level of this ruin the loots a bit wonky. 
-  30+ pml rockets is outright excessive. Especially when they're almost all in one place.
-  I'd recommend giving an actual way to use the rockets, but hide it somewhere. And cut the rocket number down to like 10 or less.
- Add a couple guns maybe? Just a bit more loot in general. Even general loot. It doesn't all have to be weaponry either. Kitchen stuff, amenities, etc.
- Spread your loot out. Seriously almost all of it is in this one area
image

Aside from all that you did excellent with this.

**Comment by Sun-Soaked** [comment]
if I could make a recommendation to the above, we have a number of excellent wrecked machines that can be salvaged for resources. Peppering them around the base could puff up the pretty meager rewards a bit

**Comment by Latentish** [comment]
Ok I'd still like you to cut down on areas a bit. At least down to 3.
I also would like you to make a ruin catalogue entry.
The ruin itself is good for merging beyond that.

**Comment by Erikafox** [comment]
I'm not really sure how, but the map catalog entry seems to already be in. Kinda wacky.

**Review by Latentish — APPROVED** [review]
merge my entire

**Pull request merged**